perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -928,6 +942,8 @@ public void setAttributes(PerunSession sess, Member member, Resource resource, L
 			}
 		}
 		removeAttributes(sess, resource, member, attributesToRemove, workWithUserAttributes);
+		//if checkAttributesSyntax fails it causes rollback so no attribute will be stored
+		checkAttributesSyntax(sess, member, resource, attributesToSet, workWithUserAttributes);
```

Comment:
Not related to this pull-request, but can you please inspect, what is this usage of weird ``Utils.startTimer()`` later in this method ? Seems like just some forgotten debug. It is only present in AttributesManagerBlImpl and just prints time, how long it took to perform specific actions (probably related to adding member). If you don't find any reasonable usage, please remove all such debug messages and then their unused variables filled by timer.

Reply:
Okay, since I haven't found any reason for those debug logs either, I have deleted them.